data/languages.yaml

```diff
@@ -0,0 +1,1514 @@
+        - un: 1
+        - una: 1
+
+
+it:
+    name: Italian
+
+    skip: ["circa", "e"]
+
+    monday:
+        - Lunedi
+    tuesday:
+        - Martedì
+    wednesday:
+        - Mercoledì
+    thursday:
+        - Giovedi
+    friday:
+        - Venerdì
+    saturday:
+        - Sabato
+    sunday:
+        - Domenica
+
+    january:
+        - Gennaio
+        - Gen
+    february:
+        - Febbraio
+        - Feb
+    march:
+        - Marzo
+        - Mar
+    april:
+        - Aprile
+        - Apr
+    may:
+        - Maggio
+        - Mag
+    june:
+        - Giugno
+        - Giu
+    july:
+        - Luglio
+        - Lug
+    august:
+        - Agosto
+        - Ago
+    september:
+        - Settembre
+        - Set
+    october:
+        - Ottobre
+        - Ott
+    november:
+        - Novembre
+        - Nov
+    december:
+        - Dicembre
+        - Dic
+
+    year:
+        - anno
+        - anni
+    month:
+        - mese
+        - mesi
+    week:
+        - settimana
+        - settimane
+    day:
+        - giorno
+        - giorni
+    hour:
+        - ora
+        - ore
+    minute:
+        - minuto
+        - minuti
+    second:
+        - secondo
+        - secondi
+
+    ago:
+        - fa
+
+    simplifications:
+        - altro ieri: 2 giorni
+        - ieri: 1 giorno
+        - oggi: 0 giorni
+
+
+pt:
+    name: Portuguese
+
+    skip: ["de", "cerca", "e", "às"]
+    pertain: ["de"]
+
+    monday:
+        - Segunda-feira
+        - Segunda
+    tuesday:
+        - Terça-feira
+        - Terça
+    wednesday:
+        - Quarta-feira
+        - Quarta
+    thursday:
+        - Quinta-feira
+        - Quinta
+    friday:
+        - Sexta-feira
+        - Sexta
+    saturday:
+        - Sábado
+    sunday:
+        - Domingo
+
+    january:
+        - Janeiro
+        - Jan
+    february:
+        - Fevereiro
+        - Fev
+    march:
+        - Março
+        - Mar
+    april:
+        - Abril
+        - Abr
+    may:
+        - Maio
+        - Mai
+    june:
+        - Junho
+        - Jun
+    july:
+        - Julho
+        - Jul
+    august:
+        - Agosto
+        - Ago
+    september:
+        - Septembro
+        - Setembro
+        - Set
+    october:
+        - Outubro
+        - Out
+    november:
+        - Novembro
+        - Nov
+    december:
+        - Dezembro
+        - Dez
+
+    year:
+        - ano
+        - anos
+    month:
+        - mês
+        - meses
+    week:
+        - semana
+        - semanas
+    day:
+        - dia
+        - dias
+    hour:
+        - hora
+        - horas
+    minute:
+        - minuto
+        - minutos
+    second:
+        - segunda
+        - segundos
+
+    ago:
+        - atrás
+        - há
+
+    simplifications:
+        - anteontem: 2 dias
+        - ontem: 1 dia
+        - hoje: 0 dias
+        - um: 1
+        - uma: 1
+        - alguns segundos: 44 segundos
+
+
+ru:
+    name: Russian
+
+    skip: ["около", "в", "и", "г."]
```

Comment:
If we'd move г. to 'hour' array it would translate the above string like 'september 2015 year 10:23' and hence freshness_parser will parse it for '2015 year' returning an incorrect date.